ui/modules/reservation.ts

```diff
@@ -312,8 +312,8 @@ export const isReservationReservable = (
       [new Date(start), subMinutes(new Date(end), 1)],
       openingHours?.openingTimes,
       activeApplicationRounds,
-      reservationBegins,
-      reservationEnds,
+      new Date(reservationBegins),
+      new Date(reservationEnds),
```

Comment:
I think you fixed this before the function was refactored, but these values also need truthiness check.